src/containers/PipelineRuns/PipelineRuns.js

```diff
@@ -150,21 +237,94 @@ export /* istanbul ignore next */ class PipelineRuns extends Component {
             lowContrast
           />
         )}
-        <Button
-          className="create-pipelinerun-button"
-          iconDescription="Create PipelineRun"
-          renderIcon={Add}
-          onClick={() => this.toggleModal(true)}
-        >
-          Create PipelineRun
-        </Button>
-        <CreatePipelineRun
-          open={this.state.showCreatePipelineRunModal}
-          onClose={() => this.toggleModal(false)}
-          onSuccess={this.handleCreatePipelineRunSuccess}
-          pipelineRef={pipelineName}
-          namespace={selectedNamespace}
-        />
+        {!validFilter && (
+          <InlineNotification
+            lowContrast
+            kind="error"
+            title={intl.formatMessage({
+              id: 'dashboard.pipelineRuns.invalidFilter',
+              defaultMessage:
+                'Filters must be of the format labelKey:labelValue and use only accepted label characters'
+            })}
+            subtitle=""
+            role="alert"
+            onCloseButtonClick={this.handleCloseFilterError}
+          />
+        )}
+        {duplicateFilter && (
```

Comment:
Too much code duplication, anyway to combine these 2  ? A duplicate filter is an invalid filter. Should invalid filter be an array or error messages instead of a boolean maybe ?